src/Components/Title.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import styled from "styled-components";
+
+const TitleStyle = styled.h1`
+color:#A0E7E5;
+cursor: default;
+display:flex;
+justify-content:center;
+height:fit-content; 
+font-display: swap;
+font-size: 2.5rem;
+font-family: 'Titan One', cursive;
+`;
+
+const Title: React.FC = () => {
+    return (
+        <TitleStyle>
+            TO-DO LIST
```

Comment:
Aqui seria bom começar a pensar em reutilização já. Você, no momento, só vai usar o título pra esse TO-DO List, mas e se quisesse usar em outro lugar? Dai, seria melhor passar a propriedade de label pra componente